mempool/clist_mempool.go

```diff
@@ -212,14 +230,15 @@ func (mem *CListMempool) Flush() {
 	mem.updateMtx.RLock()
 	defer mem.updateMtx.RUnlock()
 
-	mem.txsBytes.Store(0)
```

Comment:
Is this replaced somewhere with a write of 0?

Reply:
Yes, in `removeAllTxs`, which is guarded by `txsMtx`.